assets/src/edit-story/components/keyboard/index.js

```diff
@@ -47,12 +47,14 @@ const globalRef = createRef();
  * @param {string|Array|Object} keyNameOrSpec
  * @param {function(KeyboardEvent)} callback
  * @param {Array|undefined} deps
+ * @param {string|undefined} mode
  */
 export function useKeyDownEffect(
   ref,
   keyNameOrSpec,
   callback,
-  deps = undefined
+  deps = undefined,
+  mode = undefined
```

Comment:
`type` might make more sense for naming, as that's what Mousetrap seems to use. But that's a detail.

Reply:
We should either put it into `keyNameOrSpace` as `type`. Or we should add a separate pair of methods `useKeyUpEffect` and `useGlobalKeyUpEffect`. But I strongly prefer that `callback` and `deps` are always the last two in an effect's signature.

Reply:
Refactored